src/app/service-providers/components/InputsContainer.tsx

```diff
@@ -0,0 +1,7 @@
+type InputsContainerProps = {
+  children: React.ReactNode
+}
+
+export function InputsContainer({ children }: InputsContainerProps) {
+  return <div className="flex gap-4 flex-wrap @sm:flex-nowrap">{children}</div>
```

Comment:
To be able to use container queries, wouldn't the parent need to have an `@` prefix? 

And you can't really guarantee this context by just looking at this component, right? They're separated from each other..